### DISCUSSION TITLE
can i be a maintainer

### DISCUSSION THREAD

**Discussion by iop3** [thedev-id/thedev.id#112] [General]
Hi there, I really like this service. I wanted to be a maintainer (so that i could merge prs and stuff like that). Will you (or the team) allow that?

P.S. it's okay if you don't want me to be a maintainer, i was just wondering 😄.

**Comment by fransallen** [comment]
Thanks! I will always welcome everyone who is interested in the project, especially those who want to contribute. Right now, I'm not looking for a maintainer, but I'll let you know if I'm looking for one :)

**Reply by iop3** [reply]
ok, ping me when i could be one! thanks! @fransallen 

(sorry for late reply btw)
